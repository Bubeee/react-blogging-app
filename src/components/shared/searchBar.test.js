import React from 'react';
import * as enzyme from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import Adapter from 'enzyme-adapter-react-16';
import { SearchBar } from '.';

enzyme.configure({ adapter: new Adapter() });

describe('SearchBar', () => {
  it('should render correctly', () => {
    const output = enzyme.shallow(
      <SearchBar />
    );
    
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});