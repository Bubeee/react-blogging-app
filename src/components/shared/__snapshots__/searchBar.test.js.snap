// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBar should render correctly 1`] = `
<div
  className="search-bar__container"
>
  <input
    className="search-bar__input"
    placeholder="Type text to search"
    type="text"
  />
  <button
    className="btn btn-info btn-lg search__button"
    onClick={[Function]}
  >
    Search
  </button>
</div>
`;
